build.gradle

```diff
@@ -2,24 +2,41 @@ plugins {
     id 'org.jetbrains.kotlin.jvm' version "$kotlinVersion"
     id 'org.jetbrains.kotlin.plugin.serialization' version "$kotlinVersion"
     id 'org.openapi.generator' version "$openApiGenVersion"
+    id 'com.github.johnrengelman.shadow' version "$shadowJarVersion"
+    id 'de.undercouch.download' version "$downloadPluginVersion"
     id 'java'
     id 'application'
     id 'idea'
 }
 
-// Allow custom specification of Cineast OpenAPI config via -PapiConfigPath=...
-def apiConfigPath = project.properties["apiConfigPath"] ?: "$rootDir/cineast.json"
+// OpenAPI specifications (VREM & Cineast).
+def specPath = "$rootDir/oas"
+
+// Path for generated Cineast client sources.
+def generatedPath = "$buildDir/generated"
+
+// Path for Cineast OpenAPI specification.
+def cineastOasPath = "$specPath/cineast-oas.json"
+
+// VREM OpenAPI specification URL and local file.
+def vremOasUrl = "http://localhost:4545/swagger-docs"
+def vremOasPath = "$specPath/vrem-oas.json"
 
 // Source/Target compatibility (currently used for source & target/JVM).
 def javaCompatibility = 11
 
 // Project settings.
 group 'ch.unibas.dmi.dbis'
-version '2.1.0'
+version '3.0.0-SNAPSHOT'
```

Comment:
I guess this should rather be `3.0.0` for the PR (and release) on master

Reply:
Sure - `3.0.0` makes sense.